envs/env_4inrow.py

```diff
@@ -18,39 +18,61 @@ def __init__(self, player1, player2, board_size):
         self.player2 = player2
         self.boardH = board_size[0]
         self.boardW = board_size[1]
-        self.board = [[0] * self.boardW] * self.boardH
+        self.board = [[0 for i in range(self.boardW)] for j in range(self.boardH)]
 
     def reset(self):
-        self.board = [[0] * self.boardW] * self.boardH
+        self.board = [[0 for i in range(self.boardW)] for j in range(self.boardH)]
         return self.board
 
     def apply_action(self, player, action):
-        if action in self.available_moves(player):
-            self.board[action[0]][action[1]] = player
-            # next state, reward
-            return self.board, 0
-        return None
+        for a in self.available_moves(player):
+            if action == a[1]:
+                self.board[a[0]][a[1]] = player
+                # next state, reward
+                return self.board, 0
 
     def render(self):
+        print(" ", end="")
+        for r in range(self.boardW):
+            print("{:1d} ".format(r), end="")
+        print()
+        count = 0
         for i in self.board:
-            print(i)
+            # print("_" * (self.boardW * 2 + 1))
+            print(str(count), end="")
+            count += 1
+            for j in i:
+                print("⃒", end="")
+                if j == self.player1:
+                    # Red
+                    print("\033[31m⬤\033[30m", end="")
```

Comment:
Please use constants in the beginning of the file such as `RED="\033[31m⬤\033[30m"`.